x-pack/plugins/ml/server/routes/data_frame.js

```diff
@@ -4,18 +4,22 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import Boom from 'boom';
+
 import { callWithRequestFactory } from '../client/call_with_request_factory';
-import { wrapError } from '../client/errors';
 
 export function dataFrameRoutes(server, commonRouteConfig) {
 
   server.route({
     method: 'GET',
     path: '/api/ml/_data_frame/transforms',
-    handler(request) {
-      const callWithRequest = callWithRequestFactory(server, request);
-      return callWithRequest('ml.getDataFrameTransforms')
-        .catch(resp => wrapError(resp));
+    async handler(request) {
+      try {
+        const callWithRequest = callWithRequestFactory(server, request);
+        return await callWithRequest('ml.getDataFrameTransforms');
+      } catch (error) {
+        return Boom.badRequest(error.message || error);
```

Comment:
i believe we should be using `wrapError` here to anonymise any unexpected `500` errors that are thrown by the server.

Reply:
thanks for pointing that out, I wasn't aware of the anonymization! The backend API will change the returned error code to `400` where applicable so I reverted the whole file!